daemon/create.go

```diff
@@ -113,6 +113,10 @@ func (daemon *Daemon) containerCreate(ctx context.Context, daemonCfg *configStor
 		return containertypes.CreateResponse{Warnings: warnings}, errdefs.InvalidParameter(err)
 	}
 
+	if runtime.GOOS == "linux" && (opts.params.HostConfig.NetworkMode.IsDefault() || opts.params.HostConfig.NetworkMode.IsBridge()) && len(opts.params.HostConfig.Links) > 0 {
+		warnings = append(warnings, "Links on the default bridge network are deprecated and will be removed in a future release. You should use a custom network instead.")
```

Comment:
Perhaps slightly shorter, and remove the pronoun;

```suggestion
		warnings = append(warnings, "Links on the default bridge network are deprecated and will be removed in a future release. Use a custom network instead.")
```

What's the situation on Windows? Will we remove `--link` for the default network there as well?